docs/diagrams/ModelClassDiagram.puml

```diff
@@ -12,14 +12,19 @@ Class AddressBook
 Class ModelManager
 Class UserPrefs
 
-Class UniquePersonList
-Class Person
-Class Address
+Class UniqueStudentList
+Class Student
+Class Telegram
 Class Email
 Class Name
-Class Phone
+Class GithubUsername
+Class StudentId
 Class Tag
-
+Package lab {
```

Comment:
Very small thing, wondering if `lab` should be `Lab` instead

Reply:
Hmm, i was following how the packages are named in our project. But I just realised that even though `logic` and `model` are lowercase in our project, the original AB3 diagrams have them as `Logic` and `Model`. I think i will follow AB3 and change this to `Lab`.